backend/src/index.js

```diff
@@ -427,7 +426,9 @@ async function main() {
     await startStatsAggregation();
   }
 
-  startRegularCalculationCirculatingSupply();
+  if (nearRpc.connection.url.includes("mainnet")) {
```

Comment:
https://github.com/near/near-explorer/blob/2d71c17e09c8f379c1174ebacaa5170dc5d0c806/backend/src/config.js#L58

```suggestion
  if (wampNearNetworkName === "mainnet") {
```

Please, add the import and double check that it works fine